lib/Runtime/Base/ScriptContext.cpp

```diff
@@ -3814,11 +3848,15 @@ namespace Js
                     {
                         threadContext->GetDebugManager()->GetDebuggingFlags()->SetIsBuiltInWrapperPresent(false);
                     }
+#ifndef DISABLE_SEH
                     __try
+#endif
                     {
                         aReturn = JavascriptFunction::CallFunction<true>(function, origEntryPoint, args);
                     }
+#ifndef DISABLE_SEH
                     __finally
+#endif
                     {
                         threadContext->GetDebugManager()->GetDebuggingFlags()->SetIsBuiltInWrapperPresent(isOrigWrapperPresent);
```

Comment:
Please change these `__try/__finally` to use `TryFinally` function https://github.com/Microsoft/ChakraCore/blob/master/lib/Common/CommonPal.h#L603.